.github/workflows/build-push-images.yaml

```diff
@@ -56,6 +56,10 @@ jobs:
             vulnerability:
               - '.github/workflows/build-push-images.yaml'
               - 'ci/images/vulnerability-scan/**'
+            e2e-test-runner:
```

Comment:
Please try to keep list alphabetically ordered. It makes browsing through easier.

Reply:
fixed